content/blog/getting-started-to-generate-dscc-api-client-libraries-for-python-using-openapi-generator.md

```diff
@@ -0,0 +1,217 @@
+    list          Lists the available generators
+    meta          MetaGenerator. Generator for creating a new template set and configuration for Codegen.  The output will be based on the language you specify, and includes default templates to include.
+    validate      Validate specification
+    version       Show version information used in tooling
+
+See 'openapi-generator-cli help <command>' for more information on a specific
+command.
+```
+
+Now that the JAR file is downloaded and ready to use, lets create a Python SDK using the openAPI generator JAR file. The following command line is used for generating a Python client library using the openapi-generator.jar file.
+
+```shell
+~$ java -jar openapi-generator-cli.jar generate -i storage-api.yaml -g python -o sdks/dscc-python-sdk
+```
+
+* 'i' represents the input file, that is the OpenAPI spec which can be in the form of JSON or YAML.
+* 'generate' represents generating the code based on the specified generator
+* 'g' represents the generator/language name like Java, Go.
+* 'o' represents output directory where the client library will be generated.
+
+This Python DSCC client library can be generated in a few minutes. Below is a snippet of screen output during the generation a Python client library using the openapi-generator.jar file.
+
+![](/img/client-generation.jpg "Generating Python SDK using OpenAPI generator")
+
+The generated client library can be made available in your workstation, or it can also be uploaded to a GitHub library so that it can be made available for others to use. An example of the GitHub repository of a sample Python client library looks like this: 
+
+![](/img/python-open-api-sdk-repo.jpg "GitHub repository of Python Client library generated using OpenAPI generator")
+
+The client library comes with the following:
+
+* The code for assisting the DSCC API calls.
+* Documentation for this DSCC client library that derived from the API spec.
+* Tests codes that can be used to validate the operation of this client library.
+* Examples of every endpoint available in the README.md
+* Required python dependencies (requirements.txt and test-requirements.txt) that are required for using this SDK
+
+For instance, take a look at the README file of this repository which has the following info.
+
+1. Installation instructions
+
+![](/img/readme.png "Python SDK installation instructions")
+
+2. A sample code to get started with 
+
+![](/img/sample.png "Sample code")
+
+3. Documentation list for all endpoints
+
+![](/img/api-endpoints.png "List of endpoints")
+
+4. Documentation list for all models
+
+![](/img/models.png "List of models")
+
+5. Documentation about authorization
+
+![](/img/auth.png "Authorization of API calls")
+
+Lets run a sample code which displays the access types in DSCC. The usage of each endpoint is given under documentation of each endpoint in the form of a sample code. To execute the operations on the DSCC, all that is needed for a user to do is:
```

Comment:
Add an apostrophe to "Let's"

Rephrase beginning to either:
"Let's run a sample of code that displays the access types in DSCC."
OR
"Let's run a code sample that displays the access types in DSCC."